com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -135,6 +136,7 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             Blitter.Initialize(data.shaders.coreBlitPS, data.shaders.coreBlitColorAndDepthPS);
 
             m_BlitMaterial = CoreUtils.CreateEngineMaterial(data.shaders.blitPS);
```

Comment:
Can we remove this? If not why? I thought the point was to get rid of the old URP Blit shader?